code/modules/mob/login.dm

```diff
@@ -94,8 +95,9 @@
 	if(client)
 		if(client.view_size)
 			client.view_size.resetToDefault() // Resets the client.view in case it was changed.
-		else
-			client.change_view(getScreenSize(client.prefs.read_preference(/datum/preference/toggle/widescreen)))
+		if((client?.prefs?.read_preference(/datum/preference/toggle/widescreen)))
+			var/val = client?.prefs.read_preference(/datum/preference/numeric/icon_size)
+			INVOKE_ASYNC(client, /client.verb/SetWindowIconSize, val)
```

Comment:
if you're gonna use a temporary value like this you should give it a name that describes at least a bit what it is